src/models/entity.js

```diff
@@ -16,6 +16,9 @@ const entity = (sequelize, DataTypes) => {
         address: {
             type: DataTypes.JSON,
         },
+        latlng: {
```

Comment:
If we add the `latlng` as an array inside of the `address` then we only need to update the data json and not the schema.